src/widgets/allergies/allergies-detailed-summary.css

```diff
@@ -7,6 +7,20 @@
   padding: 0.25rem 0.5rem 0 0.5rem;
 }
 
+.allergyName {
+  font-weight: 500;
+  font-size: 1.188rem;
+}
```

Comment:
Would something like `omrs-type-title-5` be suitable? https://styleguide.openmrs.org/?path=/story/openmrs-styleguide--typography-font

Nice to avoid creating module-specific styles when we can